client/src/UpcomingRequests.js

```diff
@@ -0,0 +1,156 @@
+import React, { useState, useEffect } from 'react';
+import './App.css';
+import moment from 'moment';
+import * as api_fetch from './api'
+
+function UpcomingRequests(props) {
+
+  useEffect( () => {
+    getData();
+  }, []);
+
+  // get requests from server
+  const getData = async () => {
+   
+    api_fetch.loadAdminRequests().then((data) => {
+      props.setUser(data.map(request => {
+      
+        const a = request.date;
+        const b = [a.slice(0, 4), "-", a.slice(5, 7), "-", a.slice(8, 10)].join('');
+  
+        return (
+          {
+            title: request.firstName + ' ' + request.lastName,
+            id: request.id,
+            type: convertVacationType(request.type),
+            start: moment(b).format("D.M.YYYY"),
+            end: null,
+            status: request.status = request.status.toLowerCase()
```

Comment:
What is the variable assignment doing here? Just do `status: request.status.toLowerCase()` and call it a day.